lms/djangoapps/verify_student/urls.py

```diff
@@ -105,6 +106,13 @@
         views.ReverifyView.as_view(),
         name="verify_student_reverify"
     ),
+
+    # End point for verification status
```

Comment:
Please make sure only the GET method works on this endpoint.  We might want to explicitly restrict those and test that those restrictions are there.